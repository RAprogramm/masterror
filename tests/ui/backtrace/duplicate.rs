// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
#[error("duplicate backtrace fields")]
struct DuplicateBacktrace {
    #[backtrace]
    first: std::backtrace::Backtrace,
    #[backtrace]
    second: std::backtrace::Backtrace
}

fn main() {}
