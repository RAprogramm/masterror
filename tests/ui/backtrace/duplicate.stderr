error: multiple #[backtrace] fields are not supported
 --> tests/ui/backtrace/duplicate.rs:8:5
  |
8 |     #[backtrace]
  |     ^^^^^^^^^^^^
